INFRA-2240 - If scm tag contains github, then try to pull topics from it (#316)
jenkins-infra/update-center2: README.md

> Jenkins groups plugins into various categories in the plugin manager.

Reviewer: ```suggestion
Jenkins groups plugins into various categories in the plugin manager and on [plugins.jenkins.io](https://plugins.jenkins.io/).
```

> To set these on your plugin, add a [topic](https://help.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics) to your github repo, with the prefix jenkins-.

Reviewer: ```suggestion
To set these on your plugin, add a [topic](https://help.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics) to your github repository, with the prefix `jenkins-`.
```

For example, if you want the `matrix` label, you need to add `jenkins-matrix` to your repo's [topics](https://help.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics) (also sometimes refered to as labels).

Only labels in the [whitelist file](./src/main/resources/allowed-labels.properties) will make it into the jenkins infrastructure.

> See https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/model/Messages.properties (look for `UpdateCenter.PluginCategory`) for the localization overrides Jenkins applies.

Reviewer: ```suggestion
See https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/model/Messages.properties (look for `UpdateCenter.PluginCategory`) for the localization overrides applied by Jenkins.
```

> 

Reviewer: ```suggestion

Older plugins may have additional labels defined in the file [`label-definitions.properties`](https://github.com/jenkins-infra/update-center2/edit/master/src/main/resources/label-definitions.properties) in this repository, this approach is deprecated.

```
